parser/match_operations.go

```diff
@@ -301,8 +301,8 @@ func operationMatch(
 // equalAmounts returns an error if a slice of operations do not have
 // equal amounts.
 func equalAmounts(ops []*types.Operation) error {
-	if len(ops) <= 1 {
-		return fmt.Errorf("cannot check equality of %d operations", len(ops))
+	if len(ops) == 0 {
```

Comment:
I would've expected this test to trigger a test change...can we write a simple one for testing equal amounts with 1 op? @qiwu7

Reply:
will do

Reply:
actually there already are test cases for `equalAmounts` with only 1 op, that's why the test was failing previously.

Reply:
I have to put up a PR that touches this code, so I'm going to approve this and I'll add the test there! This PR has been back and forth enough already haha.

Reply:
cool. I can help review that when it's ready :)